pkg/cmd/delete.go

```diff
@@ -53,39 +58,43 @@ func NewDeleteCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command
 		if err != nil {
 			return err
 		}
-
-		if err := o.Complete(cmd, args); err != nil {
-			return err
+		if len(args) < 1 {
+			return errors.New("must specify name for the app")
 		}
+		o.appName = args[0]
+
 		return o.Delete()
 	}
 	return cmd
 }
 
-func (o *deleteOptions) Complete(cmd *cobra.Command, args []string) error {
-
-	if len(args) < 1 {
-		return errors.New("must specify name for the app")
-	}
-
-	namespace := o.Env.Namespace
-
-	o.Component.Name = args[0]
-	o.Component.Namespace = namespace
-	o.AppConfig.Name = args[0]
-	o.AppConfig.Namespace = namespace
-	return nil
-}
-
 func (o *deleteOptions) Delete() error {
-	o.Infof("Deleting AppConfig \"%s\"\n", o.AppConfig.Name)
-	err := o.client.Delete(context.Background(), &o.AppConfig)
+	o.Infof("Deleting AppConfig \"%s\"\n", o.appName)
```

Comment:
```suggestion
	o.Infof("Deleting App \"%s\"\n", o.appName)
```
When creating an app, we call it as `app`, so it might be better to not expose concept `AppConfig`.